src/app/frontend/deploy/deploy.scss

```diff
@@ -19,3 +19,9 @@
   min-width: 600px;
   max-width: 960px;
 }
+
+.kd-deploy-moreoptions-button {
```

Comment:
You know what? I've tested how it looks like without this style at all, at it feels really nice. "Material-like". How about removing this style entirely?

I want us to stick with material whenever possible to avoid writing our code and look like standard material app. I think there's nothing wrong if we diverge from mocks a little.